### PR TITLE
GenericScheme

### PR DESCRIPTION
**Pull request by orenyodfat**
A scheme for proposing and executing calls to an arbitrary function on a specific contract on behalf of the organization avatar.

### CODE REVIEW

File: contracts/universalSchemes/GenericScheme.sol [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+ */
+contract GenericScheme is UniversalScheme, ExecutableInterface {
+    event NewCallProposal(
+        address indexed _avatar,
+        bytes32 indexed _proposalId,
+        bytes   callData
+    );
+    event ProposalExecuted(address indexed _avatar, bytes32 indexed _proposalId,int _param);
+    event ProposalDeleted(address indexed _avatar, bytes32 indexed _proposalId);
+    event VoteOnBehalf(bytes32[] _params);
```
Comment (leviadam) [inline contracts/universalSchemes/GenericScheme.sol]: Redundant

File: test/genericscheme.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+       var testSetup = await setup(accounts,actionMock.address);
+       var callData = await createCallToActionMock(testSetup.org.avatar.address,actionMock);
+
+       var tx = await testSetup.genericScheme.proposeCall(testSetup.org.avatar.address,
+                                                           callData);
+       assert.equal(tx.logs.length, 1);
+       assert.equal(tx.logs[0].event, "NewCallProposal");
+      });
+
+           it("execute proposeCall -no decision - proposal data delete", async function() {
```
Comment (leviadam) [inline test/genericscheme.js]: Fix indentation

File: contracts/universalSchemes/GenericScheme.sol (hunk 1 above)
Comment (orenyodfat) [reply contracts/universalSchemes/GenericScheme.sol]: removed at 8f1064b

File: test/genericscheme.js (hunk 2 above)
Comment (orenyodfat) [reply test/genericscheme.js]: done at 8f1064b

**Pull request merged**